Gruntfile.js

```diff
@@ -160,6 +160,14 @@ module.exports = function (grunt) {
                     from: '<description>',
                     to: "<%=pkg.description%>"
                 }]
+            },
+            style : {
```

Comment:
@4p00rv There is a `replace` task. Do you think that it would be good if we move this inside that? Just create two different tasks inside it. One for the previous one and one for this new thing you have done here. We have been doing this for all other cases. You can check copy, server, gh-pages etc as an example